libs/core/src/lib/wizard/wizard-navigation/wizard-navigation.component.html

```diff
@@ -1,3 +1,3 @@
-<nav class="fd-wizard__navigation" [attr.aria-label]="ariaLabel">
+<nav class="fd-wizard__navigation" [attr.aria-label]="ariaLabel" role="navigation">
```

Comment:
Please explain why you need the navigation role to an element that natively has it. 
Please refer to the following screenshot taken from the https://developer.mozilla.org/ (link also provided below)

<img width="1051" alt="Screen Shot 2021-12-03 at 4 16 21 PM" src="https://user-images.githubusercontent.com/39598672/144673861-a161d073-7b4d-4482-9cc6-97142112991a.png">

Link:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role